src/scss/components/_aside.scss

```diff
@@ -47,7 +47,7 @@ aside {
 
 				li.active {
```

Comment:
Line 48-52 are identical as line line 29-33 and they are all for `li:active`. Can this section be removed?

Reply:
Yes, now it can. I have completed this.